openvdb/Grid.h

```diff
@@ -1619,11 +1707,53 @@ createLevelSet(Real voxelSize, Real halfWidth)
     return grid;
 }
 
+
+////////////////////////////////////////
+
+
+namespace internal {
+
+/// @private
+template<typename OpT, typename GridBaseT, typename T, typename ...Ts>
+struct GridApplyImpl { bool operator()(GridBaseT&, OpT&) const { return false; } };
+
+// Partial specialization for (nonempty) TypeSets
+/// @private
+template<typename OpT, typename GridBaseT, typename GridT, typename ...GridTs>
+struct GridApplyImpl<OpT, GridBaseT, TypeList<GridT, GridTs...>>
+{
+    bool operator()(GridBaseT& grid, OpT& op) const
```

Comment:
could make this a non operator so it doesn't require instances of GridApplyImpl. v minor though